Source/Core/InputCommon/ControllerEmu/StickGate.cpp

```diff
@@ -8,10 +8,50 @@
 
 #include "Common/Common.h"
 #include "Common/MathUtil.h"
+#include "Common/Matrix.h"
+#include "Common/StringUtil.h"
+#include "InputCommon/ControllerEmu/Control/Control.h"
 #include "InputCommon/ControllerEmu/Setting/NumericSetting.h"
 
+constexpr auto CALIBRATION_CONFIG_NAME = "Calibration";
```

Comment:
These constants should probably also go within the anonymous namespace below as well.

Reply:
:+1: